public/app/features/scenes/core/SceneTimeRange.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { SceneTimeRange } from './SceneTimeRange';
+
+describe('SceneTimeRange', () => {
+  it('when created should evaluate time range', () => {
+    const timeRange = new SceneTimeRange({ from: 'now-1h', to: 'now' });
+    expect(timeRange.state.value.raw.from).toBe('now-1h');
```

Comment:
As documentation, maybe would be useful to compare the whole state value as evaluated time range. I will help to understand how the evaluation transforms the passed data. Something like:
```suggestion
    expect(timeRange.state.value).toEquak({raw: {from:'now-1h', to: 'now} ...});
```